context.go

```diff
@@ -82,18 +85,18 @@ func (context *Context) start(ws *websocket.Conn) *Sender {
 }
 
 // Opens a BLIP connection to a host.
-func (context *Context) Dial(url string, origin string) (*Sender, error) {
+func (context *Context) Dial(url string, origin string, protocol string) (*Sender, error) {
```

Comment:
Preferably rename to `appProtocolId` or `subProtocol` - so that callers are clear which part of the protocol string they need to specify.